pandas/core/arrays/integer.py

```diff
@@ -95,6 +95,15 @@ def construct_array_type(cls) -> Type["IntegerArray"]:
         """
         return IntegerArray
 
+    def _get_common_type(self, dtypes: List[DtypeObj]) -> Optional[DtypeObj]:
```

Comment:
should this be common_type or common_dtype?  we've been loose about this distinction so far and i think it has caused amibiguity

Reply:
I don't care that much. I mainly used "type", because it is meant to be used in `find_common_type`.

(that `find_common_type` name is inspired on the numpy function, and that one actually handles both dtypes and scalar types, which I assume is the reason for the name. The pandas version, though, doesn't  really make the distinction, so could have been named "find_common_dtype")

Reply:
Renamed to "common_dtype" instead of "common_type". The internal function that uses this is still `find_common_type`, but that name from numpy is actually a misnomer here, since we are only dealing with dtypes, and not scalar types.

Reply:
Thanks for indulging me on this nitpick